docs/getting-started-interfaces.md

```diff
@@ -0,0 +1,45 @@
+# How do I get started, what interfaces can I use?
+
+To interact with a model it is recommend that you utilize one of the pre-existing interfaces that have been built by the open-source community.
+
+These interfaces provide several handy features:
+- A chat-like interface to interact with your model
+- Swapping between prompt types to match the model being used
+- Model re-training (currently Ooba only)
+- Support for changing model parameters on the fly
```

Comment:
I don't think they will know what all these things mean as of yet. The only thing they the only thing they've been explained is what a LLM is and how it works. I think all they really care about is "its a box that lets me talk to a LLM and train my own"

Reply:
Removed this for now as it is redundant based on other changes.